### ISSUE TITLE
Need help with catwalks

### ISSUE THREAD

**Issue by WafflesAndSyrup** [BEEmod/BEE2-items#2534]
I can't seem to get catwalks to work without putting them on top of blocks. I can't get catwalk stairs to work either.

**Comment by ZombiePwner1** [comment]
@WafflesAndSyrup The way catwalks work is abit confusing but you have to connect one with each other and it'll generate.

**Comment by LautaroL20** [comment]
@WafflesAndSyrup - To get the stair segment, you need to leave a separation of one boxel between two catwalks that are connected. Let me know if it worked.

**Comment by WafflesAndSyrup** [comment]
I tried that, not sure if i did it right, but this happened instead
![screenshot 10](https://user-images.githubusercontent.com/36249695/36933557-0555d2ea-1f16-11e8-95f1-f4823db1769d.png)
![preview0011](https://user-images.githubusercontent.com/36249695/36933559-0982ce22-1f16-11e8-8ae4-79a985ffdd13.jpg)

**Comment by cartman1337** [comment]
The distance between the bottom catwalk and the ones on the wall is too short. I believe it needs to blocks of distance per one block altitude change, so the one on the floor must be 4 blocks left of the block to blocks higher up. Also, you only need one catwalk in the mid section there, you can connect the one in the middle to both the other ones.

**Comment by cartman1337** [comment]
With a 6 block wide wall, like you have in your example, this is the best you can do, for a working stair:
![Imgur](https://i.imgur.com/njhR51l.jpg)
![Imgur](https://i.imgur.com/53R4xzG.jpg)

Two open blocks of space for each level of stairs, so for two levels up, the two sections making the stairs must be 6 blocks apart (4 open ones - 2 per height level - between them).

**Comment by ZombiePwner1** [comment]
Can work but looks weird placing it on walls.

**Comment by WafflesAndSyrup** [comment]
Does it work the same way with vactubes?

**Comment by WafflesAndSyrup** [comment]
Also sidequestion in this chamber I couldn't get it to load. 
![screenshot 11](https://user-images.githubusercontent.com/36249695/36944706-b18289c0-1fdc-11e8-8b3d-b7d577625972.png)
It gets past compiling but it gets stuck in the loading screen.

**Comment by cartman1337** [comment]
Side questions, which are actually separate issues, should be asked in their own threads. However, from experience, I can help you with that one. You have Cutout Tiles placed on the floor in various places, and they are placed wrong. When placed wrong, like that, it compiles OK, but crashes in the loading screen. You've placed them in line, and that doesn't work. The description for the item reads "Place two at opposite corners and connect with antlines. They should be on the same level, and the rectangular region should all have blocks present."
In other words, they need to be placed so that they create a rectangle between them, of at least two blocks width.
I.e.
x
  x
will work, but
x
x
will not.

**Comment by WafflesAndSyrup** [comment]
Tried it, and it still didn't load. I even removed the cutout tiles entirely but it still wouldn't load.
